code/game/objects/items/weapons/implants/implant_explosive.dm

```diff
@@ -133,6 +133,8 @@
 /obj/item/weapon/implant/dust
 	name = "duster implant"
 	desc = "An alarm which monitors host vital signs, transmitting a radio message and dusting the corpse on death."
+	icon = 'icons/effects/blood.dmi'
+	icon_state = "remains"
 
```

Comment:
That's fine for a separate PR, but I don't really want to lump together SITs and an overhaul of the action button system for all implants into the same PR.